todoapp/projects/models.py

```
@@ -1,27 +1,32 @@
-        - members (m2m field to CustomUser; create through table and enforce unique constraint for user and project)
-        - name (max_length=100)
-        - max_members (positive int)
-        - status (choice field integer type :- 0(To be started)/1(In progress)/2(Completed), with default value been 0)
 
-        Add string representation for this model with project name.
-    """
+    STATUS_CHOICES = [
+        (0, 'To be started'),
+        (1, 'In progress'),
+        (2, 'Completed'),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)